nwb.config.js

```diff
@@ -1,7 +1,14 @@
+const path = require('path');
+
 module.exports = {
   type: 'react-component',
   npm: {
     esModules: true,
-    umd: false
-  }
-}
+    umd: false,
+  },
+  webpack: {
+    extra: {
+      entry: path.resolve('./build/src'),
+    },
+  },
+};
```

Comment:
You probably don't need this configuration anymore

Reply:
Good catch!